dds/DCPS/DataReaderImpl.h

```diff
@@ -973,7 +971,24 @@ class OpenDDS_Dcps_Export DataReaderImpl
   CORBA::Long last_deadline_missed_total_count_;
   /// Watchdog responsible for reporting missed offered
   /// deadlines.
-  RcHandle<RequestedDeadlineWatchdog> watchdog_;
+  TimeDuration deadline_period_;
+  typedef OPENDDS_MULTIMAP(MonotonicTimePoint, SubscriptionInstance_rch) DeadlineQueue;
+  DeadlineQueue deadline_queue_;
+  bool deadline_queue_enabled_;
+  typedef DCPS::PmfSporadicTask<DataReaderImpl> DRISporadicTask;
```

Comment:
```suggestion
  typedef PmfSporadicTask<DataReaderImpl> DRISporadicTask;
```

And the same on lines 981-989